misc/parity/test_indexer_v_algod.py

```diff
@@ -162,7 +162,7 @@ def test_parity(reports: List[str] = ASSERTIONS, save_new: bool = True):
             old_diff = yaml.safe_load(f)
         new_diff = generate_diff(algod_swgr, indexer_swgr, excludes, diff_type)
 
-        diff_of_diffs = deep_diff(old_diff, new_diff)
+        diff_of_diffs = deep_diff(old_diff, new_diff, arraysets=True)
```

Comment:
setting `arraysets=True` results in lists of attributes which only differ by permutations as being regarded as equal.